### PR TITLE
Add NodeHealthCheck api

### PR DESCRIPTION
**Pull request by geetikakay**
Signed-off-by: Geetika Kapoor <gkapoor@redhat.com>

##### Short description:
This PR contains new api for node health check.
Also apigroup is added for node health check in resources.py

### CODE REVIEW

File: ocp_resources/node_health_check.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class NodeHealthCheck(NamespacedResource):
+    """
+    NodeHealthCheck object.
+    Reference : https://docs.openshift.com/container-platform/4.10/nodes/nodes/eco-node-health-check-operator.html
```
Comment (rnetser) [inline ocp_resources/node_health_check.py:7]: Please replace with a reference to the project (and not a 4.10 doc )

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/node_health_check.py (hunk 1 above)
Comment (geetikakay) [reply ocp_resources/node_health_check.py]: Done. Added link to the GitHub repo

**Review by rnetser — APPROVED** [review]

**Comment by geetikakay** [comment]
!verified

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Comment by myakove** [comment]
!verified

**Pull request merged**